monitor/service.go

```diff
@@ -163,28 +164,17 @@ type RemoteWriterConfig struct {
 	ClusterID  uint64
 }
 
-// SetRemoteWriter can be used via and RPC call to set a remote location
-// for writing monitoring information.
-func (m *Monitor) SetRemoteWriter(c RemoteWriterConfig) error {
-	// Ignore the monitor's config settings.
+// SetPointsWriter can be used to set a remote location for writing monitoring
+// information. This will use the default interval and database.
```

Comment:
Can this comment can be slightly altered now we're falling back to using monitor config in influx?